ApplicationInsights not working

### Expected behaviour

Skill usage is reported to ApplicationInsights.

### Actual behaviour

Requests to the skill's Lambda function fail with this error:

```
{
  "errorMessage": "Cannot find module 'diagnostic-channel'",
  "errorType": "Error",
  "stackTrace": [
    "Function.Module._load (module.js:276:25)",
    "Module.require (module.js:353:17)",
    "require (internal/module.js:12:17)",
    "Object.<anonymous> (/var/task/node_modules/applicationinsights/out/AutoCollection/CorrelationContextManager.js:4:28)",
    "Module._compile (module.js:409:26)",
    "Object.Module._extensions..js (module.js:416:10)",
    "Module.load (module.js:343:32)",
    "Function.Module._load (module.js:300:12)",
    "Module.require (module.js:353:17)",
    "require (internal/module.js:12:17)",
    "Object.<anonymous> (/var/task/node_modules/applicationinsights/out/applicationinsights.js:2:33)",
    "Module._compile (module.js:409:26)",
    "Object.Module._extensions..js (module.js:416:10)",
    "Module.load (module.js:343:32)",
    "Function.Module._load (module.js:300:12)"
  ]
}
```

### Steps to reproduce

Invoke the skill.

See #41, #43 and #44.

--- a/.travis.yml
+++ b/.travis.yml
@@ -16,6 +16,9 @@ branches:
     - master
     - deploy
 
+before_deploy:
+  - zip -r alexa-london-travel.zip .
+
 deploy:
   - provider: lambda
     function_name: "alexa-london-travel-dev"
@@ -24,6 +27,7 @@ deploy:
     runtime: "nodejs6.10"
     timeout: 10,
     handler_name: "handler"
+    zip: "alexa-london-travel.zip"
     access_key_id:
       secure: "aJjugFDE24Z5UB/3PEGJEOlmpLFTUEQvtrczJiRRVBd1cG535MVmSm2GlVzVk1J96yqoVc8VqSEgWcdqlMcU4jupd6BxxFYDZE8k8hEQ9KmIFqG+VdBqn6SskadDVi/sihDakMvGVLi9koVOPfbYPoxQn+YOh3Ohu9RFbGKlUaX1/7s29VoBsX/e2iWvJQDVGqMWMg1hMtf91diOuxLbiUgsZWHQppm34UiM2oJBFHLuCjvSbtJMWJxEEULou9dgip1bCj5IxrpIM68E3Mn9ttK4ZotyL3LsgJYHp9P6KapzytKSa5eBoSXhJZx1FWMBSBW258uOtByARQyKBKrPg+csS7H8kFLNnaWLQhJqQGjU9eex6eVRDdeTcp4z5yfeGG803CrvWGcyHXANI/s1H0ktdzNK4T/SWEDh4oTgZzjaJdF15Upgw2GczO5TeO+9kbqc8cBHPHZmWp6q+bb6svvlX+EU3brBCcCIHDkwAp4Yv4rT/41ZKh0aZJ61/1Ulkb/xDcZh30kV8gc3dqnF6L8fhDXIfL1wMvw88lItiHiADcaqVV2Bb4OT4vP+cuSx7Ys7e7eyJn2y4qMqeCiFaGKqqYTbPG1gXJCLIhlqwLYJsf3yweTar8qqnrMhSJ5D6lRp9Eh2xoAk60jIBJgCH/oRjp0aErxJ4/iwxZcNx0Q="
     secret_access_key:
@@ -37,6 +41,7 @@ deploy:
     runtime: "nodejs6.10"
     timeout: 10,
     handler_name: "handler"
+    zip: "alexa-london-travel.zip"
     access_key_id:
       secure: "aJjugFDE24Z5UB/3PEGJEOlmpLFTUEQvtrczJiRRVBd1cG535MVmSm2GlVzVk1J96yqoVc8VqSEgWcdqlMcU4jupd6BxxFYDZE8k8hEQ9KmIFqG+VdBqn6SskadDVi/sihDakMvGVLi9koVOPfbYPoxQn+YOh3Ohu9RFbGKlUaX1/7s29VoBsX/e2iWvJQDVGqMWMg1hMtf91diOuxLbiUgsZWHQppm34UiM2oJBFHLuCjvSbtJMWJxEEULou9dgip1bCj5IxrpIM68E3Mn9ttK4ZotyL3LsgJYHp9P6KapzytKSa5eBoSXhJZx1FWMBSBW258uOtByARQyKBKrPg+csS7H8kFLNnaWLQhJqQGjU9eex6eVRDdeTcp4z5yfeGG803CrvWGcyHXANI/s1H0ktdzNK4T/SWEDh4oTgZzjaJdF15Upgw2GczO5TeO+9kbqc8cBHPHZmWp6q+bb6svvlX+EU3brBCcCIHDkwAp4Yv4rT/41ZKh0aZJ61/1Ulkb/xDcZh30kV8gc3dqnF6L8fhDXIfL1wMvw88lItiHiADcaqVV2Bb4OT4vP+cuSx7Ys7e7eyJn2y4qMqeCiFaGKqqYTbPG1gXJCLIhlqwLYJsf3yweTar8qqnrMhSJ5D6lRp9Eh2xoAk60jIBJgCH/oRjp0aErxJ4/iwxZcNx0Q="
     secret_access_key:

